scripts/kubectl-workspace-ensure

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
```

Comment:
FYI, the name of this plugin is a bit nerve-wracking, since `kubectl workspace` is already claimed by kcp.  I think that kcp does not have a `kubectl workspace ensure` subsubcommand, so I think this will work.